css/notes.css

```diff
@@ -19,7 +19,9 @@
 #app-navigation .active a {
 	padding-right: 70px;
 }
-
+#app-navigation > ul > li.note.has-error a{
+    color:#f00;
+}
```

Comment:
This doesn't work for me, since the `<li>` has not the class `note`. The CSS selector should be `#app-navigation > ul > li.has-error a` instead.